app/uk/gov/hmrc/hec/testonly/models/SaveTaxCheckRequest.scala

```diff
@@ -35,7 +35,8 @@ final case class SaveTaxCheckRequest(
   taxCheckStartDateTime: ZonedDateTime,
   isExtracted: Boolean,
   source: HECTaxCheckSource,
-  relevantIncomeTaxYear: Option[TaxYear]
+  relevantIncomeTaxYear: Option[TaxYear],
+  correctiveAction: Option[CorrectiveAction]
```

Comment:
I think we can avoid adding correctiveAction here , we can just set it to None in implementation. @AndyHWChung should we allow QA to add correctiveAction through request?

Reply:
It's optional, so it'll get set to `None` if they don't specify it

Reply:
it adds flexibility for the future if they want to use it for some reason in the future I think - if shouldn't break anything now cos it's optional. We should let the QA's know that this is an option so they know about it.